HarmonyOS/Day5 Assingment/Sanku Yogesh/App5/entry/src/main/java/com/example/myapplication/database/User.java

```diff
@@ -0,0 +1,78 @@
+package com.example.myapplication.database;
+
+import ohos.data.orm.OrmObject;
+import ohos.data.orm.annotation.Entity;
+import ohos.data.orm.annotation.Index;
+import ohos.data.orm.annotation.PrimaryKey;
+
+@Entity(tableName = "user",
+        indices = {@Index(value = {"email", "password"}, name = "name_index", unique = true)})
+public class User extends OrmObject {
```

Comment:
Please clean up the code before commit and remove unused functions - Make it practice always.